lib/models/watcher.js

```diff
@@ -105,17 +105,17 @@ Watcher.detectWatcher = function(ui, _options) {
       debug('detected watchman: %s', version);
 
       var semver = require('semver');
-      if (semver.satisfies(version, '^3.0.0')) {
-        debug('watchman %s does satisfy: %s', version, '^3.0.0');
+      if (semver.satisfies(version, '>= 3.0.0 < 5.0.0')) {
+        debug('watchman %s does satisfy: %s', version, '>= 3.0.0 < 5.0.0');
```

Comment:
Per https://github.com/ember-cli/ember-cli/pull/5026#issuecomment-153366455, please change this to `>= 3.0.0`.